examples/cox.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+"""Cox process model using mean-field variational inference.
+   Ref: Miller et al, 2014.
```

Comment:
does andrew miller et al's work use a form of black box variational inference? otherwise it may be better to cite a more classical cox process paper.